test/systemtest/earlgrey/test_sim_verilator.py

```diff
@@ -276,3 +276,54 @@ def test_spiflash(tmp_path, bin_dir):
 
     sim.terminate()
 
+
+def test_openocd_basic_connectivity(tmp_path, bin_dir, topsrcdir, openocd):
+    """Test the basic connectivity to the system through OpenOCD
+
+    Run earlgrey in Verilator, connect to it via OpenOCD, and check that the
+    JTAG TAP, the harts, etc. are found as expected. No further interaction
+    with the core or the system (bus) is performed.
+    """
+    # Run a simulation (bootrom only, no app beyond that)
+    sim_path = bin_dir / "hw/top_earlgrey/Vtop_earlgrey_verilator"
+    rom_elf_path = bin_dir / "sw/device/boot_rom/boot_rom_sim_verilator.elf"
+    sim = VerilatorSimEarlgrey(sim_path, rom_elf_path, tmp_path)
```

Comment:
note (partially to myself) that this will have to be extended to include the `otp_img` path as well, see: https://github.com/lowRISC/opentitan/pull/5364 

(depends a bit which PR lands first)

Reply:
I won (because I didn't review your PR first :-)) I'll take a look at that tomorrow, it's on my list.

Reply:
Haha alright, I'll rebase it :)

Reply:
Thanks! I also took a couple more minutes to review your PR, wasn't as long as I initially feared. nice work!

Reply:
Thanks. Yes, fortunately the mechanisms we already had in place for ROM loading where a pretty good fit for what we needed for OTP.